modals/cc-modal.html

```diff
@@ -0,0 +1,11 @@
+<div class="modal-header" xmlns="http://www.w3.org/1999/html">
+    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+    <h3>Credit Card Security Policy Modal header</h3>
+</div>
+<div class="modal-body">
+    <p><pre>I will be careful with credit cards.</pre></p>
```

Comment:
It's not, no. :-)

This file is just for my local testing; the final modal is [this](https://staffweb.cru.org/ucmcont/fragments/frag_sw_assets/piu-intercept/affirmation-cc.html).